packages/table/src/cell/cell.tsx

```diff
@@ -19,6 +19,15 @@ export interface ICellProps extends IIntentProps, IProps {
     style?: React.CSSProperties;
 
     /**
+     * If true, the cell will be rendered above overlay layers to enable mouse
+     * interactions within the cell.
+     *
```

Comment:
don't need the blank line here (see `tooltip` below) but if you like it then whatevs